student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
+    @classroom = classroom
+    super(id, age: age, name: name, parent_permission: parent_permission)
```

Comment:
👍 Good job on your implementation! I appreciate your efforts in working with the `person.rb` file, and it seems to be functioning correctly. However, I have a few suggestions for you to consider:

- [ ] It's great that you've tested the `person.rb` file, but I recommend trying out the other files as well to ensure that you can create students and teachers successfully.

- [ ] In the `person` class, you're generating an ID. However, when you call the parent class with `super` from the `student` and `teacher` classes, you're passing the wrong number of arguments. The `id` should not be sent to the person class as it is generated there, and there is no parameter for the ID value in the person class. Please update your code in both the `student` and `teacher` classes to address this issue.

By making these adjustments, you'll ensure that your code correctly follows the inheritance structure and creates students and teachers as intended. Keep up the great work! 👍📝